std/algorithm/iteration.d

```diff
@@ -4257,6 +4267,11 @@ struct Permutations(Range)
 
         next(2);
     }
+
+    @property bool empty() pure nothrow @safe
```

Comment:
Not sure what this change is doing in this PR, but attributes for member functions of type templates are inferred, so explicit attributes are not necessary here (also `@nogc` is conspicuous by absence)